pkg/cloudprovider/gcp/object_store.go

```diff
@@ -194,11 +215,37 @@ func (o *ObjectStore) DeleteObject(bucket, key string) error {
 	return errors.Wrapf(o.client.Bucket(bucket).Object(key).Delete(context.Background()), "error deleting object %s", key)
 }
 
+func (o *ObjectStore) SignBytes(bytes []byte) ([]byte, error) {
+	name := "projects/-/serviceAccounts/" + o.googleAccessID
+	payload := strings.Builder{}
+	if _, err := payload.Write(bytes); err != nil {
+		return nil, err
+	}
+	resp, err := o.iamSvc.Projects.ServiceAccounts.SignBlob(name, &iamcredentials.SignBlobRequest{
+		Delegates: []string{name},
```

Comment:
I dont' think you need to specify the `name` here (thats only used normally when you chain different service accounts together

Reply:
I can confirm that everything works fine without L228, and keeping it requires additional permissions, so let's remove.